examples/menubar/menubar-navigation.html

```diff
@@ -95,7 +109,7 @@ <h2 id="ex_label">Example</h2>
                 <a role="menuitem" href="mb-admissions.html#apply">Apply</a>
               </li>
               <li role="none">
-                <a role="menuitem" href="#" aria-haspopup="true" aria-expanded="false">Tuition</a>
+                <a role="menuitem" href="" aria-haspopup="true" aria-expanded="false">Tuition</a>
                 <ul role="menu" aria-label="Tuition Information">
```

Comment:
For consistency, the menu's aria-label should be the same as the menuitem text.
i.e. After opening the "Tuition" menuitem, I expected to hear "Tuition menu", but instead heard "Tuition Information menu".

```suggestion
                <ul role="menu" aria-label="Tuition">
```